File: PSP/Proyectos/LoginSpringAndroid_AdrianSaavedra/LoginSpring_AdrianSaavedra/src/main/java/org/example/loginspring_adriansaavedra/ui/utils/AuthenticationResponse.java
package org.example.loginspring_adriansaavedra.ui.utils;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.example.loginspring_adriansaavedra.common.Constantes;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class AuthenticationResponse {
    @JsonProperty(Constantes.ACCESS_TOKEN)
    private String accessToken;
    @JsonProperty(Constantes.REFRESH_TOKEN)
    private String refreshToken;
}

File: MOBILES/PlayersApp_AdrianSaavedra/app/src/main/java/com/example/playersapp_adriansaavedra/data/remote/utils/AuthenticationResponse.kt
package com.example.playersapp_adriansaavedra.data.remote.utils

import com.example.playersapp_adriansaavedra.ui.Constantes
import com.google.gson.annotations.SerializedName

data class AuthenticationResponse(
    @SerializedName(Constantes.AT_JSON)
    val accessToken: String,
    @SerializedName(Constantes.RT_JSON)
    val refreshToken: String,
)